app/models/reporting/workload_report.rb

```diff
@@ -8,7 +8,9 @@ class WorkloadReport
     # combined into the final row. For example:
     #   "0 days", "1 day", "2 days", "Between 3 and 9 days"
     #
-    def initialize(observed_at: Time.current, age_limit: 9, number_of_rows: 5)
+    def initialize(observed_at: Time.current, age_limit: 9, number_of_rows: 5,
```

Comment:
Just a thought is there a concept hiding here `ReportConfig` maybe - just an idea but probs not something to change in this PR